src/Util/ConfigFileLoader.php

```diff
@@ -83,10 +90,14 @@ class ConfigFileLoader
 	 */
 	private $staticDir;
 
-	public function __construct(string $basePath)
+	public function __construct(string $basePath, array $server)
```

Comment:
I feel like we should only provide the config dir, and pull the path determination logic out of this particular class